docs/cloud/automation/terraform.md

```diff
@@ -1,2 +1,244 @@
+---
 
+# Terraform provider 
+
+## Installation
+
+The current version of the provider is : {{ $frontmatter.terraform_current_version }}.
+
+The releases are available in Terraform's official [registry][terraform registry] and via [GitHub releases][terraform github releases].
+
+### Terraform 0.13+
+
+Terraform supports third party modules installed via the plugin registry.  
+Add the following to your terraform module configuration.
+
+
+<<< @/docs/cloud/automation/snippets/providers_eventstore.tf.hcl
+
+### Terraform 0.12
+
+In order for Terraform to find the plugin, the appropriate binary must be placed into the Terraform third-party plugin directory.  
+The location varies by operating system:
+
+- Linux and macOS `~/.terraform.d/plugins`
+- Windows `%APPDATA%\terraform.d\plugins`
+
+Alternatively, the binary can be placed alongside the main `terraform` binary.
+
+You can download the provider using the following commands:
+
+- Linux:
+
+<<< @/docs/cloud/automation/snippets/download_provider_linux.sh
+ 
+- macOS:  
+
+<<< @/docs/cloud/automation/snippets/download_provider_macos.sh
+
+- Windows (PowerShell): 
+
+<<< @/docs/cloud/automation/snippets/download_provider_windows.ps1.powershell
+
+## From Source 
+
+If you prefer to install from source, use the `make install` target in this [repository][terraform github].
+
+You will need a Go 1.13+ development environment.
+
+## Provider Configuration
+
+The Event Store Cloud provider must be configured with an access token.   
+There are several additional options that may be useful.
+
+Provider configuration options are:
+
+| Option              | Environment Variable   | Description                                                                                                          |
+| :------------------ | :--------------------- | :------------------------------------------------------------------------------------------------------------------- |
+| `token`             | `ESC_TOKEN`            | *Required*, your access token for Event Store Cloud                                                                  |
+| `organization_id`   | `ESC_ORG_ID`           | *Required*, the identifier of your  Event Store Cloud organization.                                                  |
+| `url`               | `ESC_URL`              | *Optional*, the URL of the Event Store Cloud API. This defaults to the public cloud instance of Event Store Cloud    |
+| `token_store`       | `ESC_TOKEN_STORE`      | *Optional*, the location on the local filesystem of the token cache. This is shared with the Event Store Cloud CLI.  |
+                                                                                                                                                                       
+### Obtaining the `token`
+Currently this token must be created and displayed with the [esc cli][esc cli github releases] tool.  
```

Comment:
**SUGGESTION:** It might be better to link to the Github repository. If someone is not aware of the esc cli tool, they could read the readme first before going to the release section. Might be worth not using shortcut name but full `EvenStoreDB Cloud CLI`